_news/2019-10-22-pythia-renamed/en.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Pythia judge renamed to Python"
+description: "Pythia judge renamed to Python"
+slug: "pythia-renamed"
+date: "2019-10-22"
+lang: en
+---
+
+To avoid confusion and to be more consistent with other judges, the Pythia judge has been renamed to Python. This change was made automatically and requires no further action from users. In addition, the Biopythia judge was merged into the Python judge, since they used the same codebase. All python exercises can now just use the python judge.
```

Comment:
```suggestion
To avoid confusion and to be more consistent with other judges, the `pythia` judge has been renamed to `python`. This change was made automatically and requires no further action from users. In addition, the `biopythia` judge was merged into the Python judge, since they used the same codebase. All python exercises can now just use the `python` judge.
```